services/lgtm/lgtm.tester.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('../service-tester');
+
+const t = new ServiceTester({ id: 'lgtm', title: 'LGTM' })
+module.exports = t;
+
+t.create('total alerts for a project')
+  .get('/alerts/g/apache/cloudstack.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'lgtm',
+    value: Joi.string().regex(/^[0-9kM.]+ alerts?$/)
```

Comment:
I think this regex should allow 'alert' or 'alerts' (i.e: it should pass however many alerts `apache/cloudstack` has when we run the test).

Reply:
Sorry - ignore this comment. I've just noticed it is already `alerts?$` not `alerts$`.
My bad

Reply:
:+1: In fairness, i did originally write it with out the `?`, i'll add tests to make sure that it prints it our correctly